src/main.cpp

```diff
@@ -5693,23 +5703,44 @@ bool TallyResearchAverages_v9()
     if(fDebug) printf("TallyResearchAverages: start %d end %d\n",nMaxDepth,nMinDepth);
 
     mvMagnitudesCopy.clear();
-    int iRow = 0;
-
     CBlockIndex* pblockindex = pindexBest;
     if (!pblockindex)
     {
         bNetAveragesLoaded = true;
         return true;
     }
+
+    // Seek to head of tally window.
     while (pblockindex->nHeight > nMaxDepth)
     {
         if (!pblockindex || !pblockindex->pprev || pblockindex == pindexGenesisBlock) return false;
         pblockindex = pblockindex->pprev;
     }
 
-    if (fDebug3) printf("Max block %f, seektime %f",(double)pblockindex->nHeight,(double)GetTimeMillis()-nStart);
+    if (fDebug3) printf("Max block %i, seektime %" PRId64, pblockindex->nHeight, GetTimeMillis()-nStart);
```

Comment:
Heights are int, not 64. On 64bit is equal, now warning.

Reply:
Odd, I don't get a warning on a 64bit system. The `PRId64` is for the time which is 64bit. What's the warning?